ledger-core/v2/network/facade/facade.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/insolar/blob/master/LICENSE.md.
+
+package facade
+
+import (
+	"context"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/bus"
+	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/v2/pulse"
+
+	"github.com/pkg/errors"
+)
+
+type Marshaler interface {
```

Comment:
this one should be defined somewhere near payload, not here, so it would be easier to update it in one place